benchmarks/build.gradle

```diff
@@ -39,12 +39,9 @@ dependencies {
             project(':grpc-interop-testing'),
             libraries.junit,
             libraries.mockito,
-            libraries.hdrhistogram
-    if (osdetector.os == "linux") {
-        // These are only available on linux.
-        compile libraries.netty_tcnative,
-                libraries.netty_transport_native_epoll
-    }
+            libraries.hdrhistogram,
+            libraries.netty_tcnative,
```

Comment:
It appears this is available for OSX, linux, and windows. native-epoll, however is only available for linux.